### ISSUE TITLE
Twitter verify in bio option and/or retweet validation periodicially

### ISSUE THREAD

**Issue by amicone** [keybase/keybase-issues#1670]
Some of us use tools that regularly delete tweets from our feeds after a certian amount of time. It keeps twitter feeds clean and current and gives trolls less stuff to dig back through. However, that doesn't work well when the verify twitter feature depends on that tweet staying up. Is there a way to have it repost the verification tweet at a regular interval and/or just have some verifying information within the bio instead of a tweet?

**Comment by chigh** [comment]
You'd just have to reprove your Twitter account by posting a new proof.

**Comment by amicone** [comment]
Guess I need to put a reoccurring item in my to-do list manager to re-prove every two weeks, since that's how fast my tweets roll off.

**Comment by chigh** [comment]
Do you have something that auto deletes them or is it a manual process? If it's manual, why can't you leave that one tweet? I'm just curious as to why you'd want to continuously have to do that.

**Comment by amicone** [comment]
Yes, it auto-deletes them every two weeks, it requires no action on my part and someone else wrote it. The reasons for doing it are mainly message shaping and minimizing digital footprint. I do some research on privacy engineering and social presence, and so far I've proved out only that despite people archive tweets all over the place that people are lazy and don't go looking for what's not there. I'll send you a link if I ever get an article out of it. :)

I already figured out a way to automatically repost it so I'm covered for this, but food for thought.
